packages/flutter_tools/lib/src/commands/build_ios_framework.dart

```diff
@@ -367,15 +371,17 @@ end
     final Status status = globals.logger.startProgress(
       ' ├─Assembling Flutter resources for App.framework...', timeout: timeoutConfiguration.slowOperation);
     try {
+      if (buildInfo.mode == BuildMode.debug) {
       await _bundleBuilder.build(
```

Comment:
This is just doing extra work now. Once there is only a single assemble call it can go away